### PR TITLE
https://issues.redhat.com/browse/ACM-11617

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

**Review by zlayne — APPROVED** [review]

File: observability/insights_intro.adoc [hunk 1]
```diff
@@ -61,6 +61,19 @@ You can view the identified issues on a specific cluster. Complete the following
...
 .. Locate the `source` parameter, which informs you of the component that sent the violation. The value options are `grc` and `insights`.
 
+[#update-risks]
+=== Viewing update risk predictions
+
+View the potential risks of updating your managed clusters. Complete the following steps:
+
+. Log in to your managed cluster. 
+. Go to the _Overview_ page. 
+. From the _Powered by Insights_ section, you can view the percentage of clusters with predicted risks, which are listed by severity.
```
Comment (swopebe) [inline observability/insights_intro.adoc:71]: changed this a bit for conciseness (line 71), clickable items should be bold, made a couple changes there.

File: observability/insights_intro.adoc [hunk 2]
```diff
@@ -61,6 +61,19 @@ You can view the identified issues on a specific cluster. Complete the following
...
+. Select the number for the severity to view the list of clusters from the _Clusters_ page.
+. Select the cluster that you want, then click the *Actions* drop-down button.
+. Click *Upgrade clusters* to view the risk for the upgdate.
+. Click the link in the _Upgrade risks_ column from the _Upgrade clusters_ modal. Information is displayed from the Hybrid Cloud console.
```
Comment (swopebe) [inline observability/insights_intro.adoc]: This last line is confusing, seems we can change it around???

`Click the link in the _Upgrade risks_ column from the _Upgrade clusters_ modal. Information is displayed from the Hybrid Cloud console.`

`From the _Upgrade clusters_ modal, find the  _Upgrade risks_ column and click the link `_(though what is this link--a link to the console?)_ `to view information in the Hybrid Cloud console. `(something like this)

File: release_notes/whats_new.adoc [hunk 3]
```diff
@@ -51,6 +51,8 @@ For other Application topics, see link:../applications/app_management_overview.a
 
 * You can customize the Observatorium API and Alertmanager URLs. See link:../observability/customize_observability.adoc#custom-obervatorium-alert-url[Customizing the managed cluster Observatorium API and Alertmanager URLs].
 
+* You can view the update risks that are present for your managed clusters by enabling the _Fleet_ switch. See, link:../observability/insights_intro.adoc#update-risks[Viewing update risk predictions].
```
Comment (swopebe) [inline release_notes/whats_new.adoc]: `You can now enable the _Fleet_ switch to view update risk predictions for your managed cluster.`

The only other thing is I would have liked to see why I want to do this in the intro and what's new. What risks? Just another line/word ortwo about the benefit of this function, especially in the What's new.

**Review by swopebe — APPROVED** [review]

File: observability/insights_intro.adoc (hunk 2 above)
Comment (dockerymick) [reply observability/insights_intro.adoc]: The number of risks is hyperlinked

Comment (dockerymick) [inline observability/insights_intro.adoc]:
```suggestion
. From the _Upgrade clusters_ modal, find the _Upgrade risks_ column and click the link for the number of risks to view information in the Hybrid Cloud console.
```

File: release_notes/whats_new.adoc (hunk 3 above)
Comment (dockerymick) [inline release_notes/whats_new.adoc]:
```suggestion
* You can enable the _Fleet_ switch to view the update risks that are present for your managed clusters. See, link:../observability/insights_intro.adoc#update-risks[Viewing update risk predictions]. 
```

File: release_notes/whats_new.adoc [hunk 4]
```diff
@@ -51,6 +51,8 @@ For other Application topics, see link:../applications/app_management_overview.a
 
 * You can customize the Observatorium API and Alertmanager URLs. See link:../observability/customize_observability.adoc#custom-obervatorium-alert-url[Customizing the managed cluster Observatorium API and Alertmanager URLs].
 
+* You can enable the _Fleet_ switch to view the update risks that are present for your managed clusters. See, link:../observability/insights_intro.adoc#update-risks[Viewing update risk predictions].
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]:
```suggestion
* You can enable the _Fleet_ switch to view the potential risks for updating your managed clusters. See, link:../observability/insights_intro.adoc#update-risks[Viewing update risk predictions]. 
```

File: observability/insights_intro.adoc [hunk 5]
```diff
@@ -61,6 +61,19 @@ You can view the identified issues on a specific cluster. Complete the following
 .. Select the _Status_ tab and click the *View details* link to view the `PolicyReport` YAML file.
 .. Locate the `source` parameter, which informs you of the component that sent the violation. The value options are `grc` and `insights`.
 
+[#update-risks]
+=== Viewing update risk predictions
+
+View the potential risks of updating your managed clusters. Complete the following steps:
```
Comment (dockerymick) [inline observability/insights_intro.adoc]:
```suggestion
View the potential risks for updating your managed clusters. Complete the following steps:
```

**Pull request merged**